examples/lora/sensor-lorawan/Makefile

```diff
@@ -0,0 +1,36 @@
+# Makefile for user application
+
+# Specify this directory relative to the current application.
+TOCK_USERLAND_BASE_DIR = ../../../
+
+STACK_SIZE = 4096
+
+# Which files to compile.
+CXX_SRCS := $(wildcard *.cc)
+
+# Include the core RadioLib headers
+override CPPFLAGS += -isystem $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/src
+
+# Include the Tock specific headers
+override CPPFLAGS += -isystem $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/examples/NonArduino/Tock
+
+# Include the base of libtock-c to fix the libtock/ includes from RadioLib
+override CPPFLAGS += -I$(TOCK_USERLAND_BASE_DIR)/ -DRADIOLIB_CLOCK_DRIFT_MS=9
+
+ifneq ($(CI),)
+override CPPFLAGS += "-DRADIO_CONFIG_CI=radioConfig_example.h"
+endif
+
+# Use the libtock-c Make system
+LIBS_cortex-m0 += $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/build/cortex-m0/RadioLib.a
+LIBS_cortex-m3 += $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/build/cortex-m3/RadioLib.a
+LIBS_cortex-m4 += $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/build/cortex-m4/RadioLib.a
+LIBS_cortex-m7 += $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/build/cortex-m7/RadioLib.a
+
+LIBS_rv32imc += $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/build/rv32imc/RadioLib.a
+LIBS_rv32imac += $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib/build/rv32imac/RadioLib.a
+
+EXTERN_LIBS := $(TOCK_USERLAND_BASE_DIR)/libradio/RadioLib
+include $(TOCK_USERLAND_BASE_DIR)/libradio/Makefile
```

Comment:
This `include` should happen as a result of setting `EXTERN_LIBS`.

Reply:
It doesn't get pulled in correctly as we need to use `libradio/RadioLib` for the `EXTERN_LIBS`

Reply:
The library makefile should adjust the paths so that all an app developer needs to do is EXTERN_LIBS. Our other libraries (eg openthread and u8g2) are good examples of how this can be done.